### DISCUSSION TITLE
closed: buffer length exceeds 64

### DISCUSSION THREAD

**Discussion by fflyerm** [bluenviron/mediamtx#3554] [Questions]
### Question

Hi all, 
I use an Pi 5 with mediamtx. Want to stream from an obs software like Streamlabs to the local rtsp Server. Also tried othe obs software. I got on the pi this message: closed: buffer length exceeds 64
Ist there something I could change in the yml file that this works?

Thanks& regards,
Michael

**Comment by fflyerm** [comment]
why closed?

**Comment by aler9** [comment]
this can't be answered without instructions on how to replicate the problem. Create another question and include, as mentioned in the instructions:

* the server version you are using
* precise instructions on how to replicate the problem
* server logs with setting logLevel set to debug
